### DISCUSSION TITLE
Test should have ability to  extracting data from last responses and chaining requests

### DISCUSSION THREAD

**Discussion by visonforcoding** [Kong/insomnia#3998] [Ideas]
now i want to test a suite.

1. Pepole submit a order
2. Sales take this order  and change order status

api 2 need api 1 this reponse body .

because api 2 need a order_no

**Comment by develohpanda** [comment]
Thanks for this request; I agree, having the ability to change the request body (and headers) when sending a request via unit tests is needed!

**Reply by visonforcoding** [reply]
yes, expect these features to appear in the new version . thanks for your work. i love  insomnia .
